obspy/clients/nrl/client.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Client for accessing the Nominal Response Library (http://ds.iris.edu/NRL/).
+
+:copyright:
+    Lloyd Carothers IRIS/PASSCAL, 2016
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import codecs
+import io
+import os
+import sys
+import warnings
+
+import requests
+
+import obspy
+from obspy.core.compatibility import configparser
+from obspy.core.inventory.response import ResponseStage
+from obspy.core.inventory.util import _textwrap
+
+
+# Simple cache for remote NRL access. The total data amount will always be
+# fairly small so I don't think it needs any cache eviction for now.
+_remote_nrl_cache = {}
+
+def _cleanup_response(response):
```

Comment:
I'd be fine with removing this again but what exactly is the rationale for keeping this? The function only removes a stage if it really does absolutely nothing so in a sense it get's rid of "response noise" distracting from the actual response definition.